core/src/main/scala/org/apache/spark/sql/catalyst/catalog/TiCompositeSessionCatalog.scala

```diff
@@ -170,7 +170,7 @@ class TiCompositeSessionCatalog(val tiContext: TiContext)
   override def listTables(db: String): Seq[TableIdentifier] = listTables(db, "*")
 
   override def listTables(db: String, pattern: String): Seq[TableIdentifier] = {
-    val currentSessionCatalog = catalogOf(Some(db)).getOrElse(throw new NoSuchDatabaseException(db))
```

Comment:
Why change from getOrElse to get?

Reply:
because it is useless. getOrElse is done in catalogOf.